docs/_data/icons.json5

```diff
@@ -575,4 +541,300 @@
             'person',
         ],
     },
+
+    /* DEPRECATED */
+    /* TODO: migrate to separate file */
```

Comment:
TODO: update icons vue demo to grab an additional file (if deprecated icons should be shown)